### PR TITLE
[merged] More glnx/gfile porting

### PR DESCRIPTION
**Pull request by cgwalters**
:train:

### CODE REVIEW

File: src/libostree/ostree-repo-static-delta-core.c [hunk 1]
```diff
@@ -159,7 +159,8 @@ ostree_repo_list_static_delta_names (OstreeRepo                  *self,
     }
 
   ret = TRUE;
-  gs_transfer_out_value (out_deltas, &ret_deltas);
+  if (out_deltas)
+    *out_deltas = g_steal_pointer (&ret_deltas);
```
Comment (jlebon) [inline src/libostree/ostree-repo-static-delta-core.c]: Should we work towards phasing out `ot_transfer_out_value` as well in favour of this?

File: src/libotutil/ot-fs-utils.c [hunk 2]
```diff
@@ -230,3 +230,24 @@ ot_openat_ignore_enoent (int dfd,
  out:
   return ret;
 }
+
+GBytes *
+ot_file_map_contents_bytes (int dfd,
```
Comment (jlebon) [inline src/libotutil/ot-fs-utils.c]: May  I suggest a `_at` suffix or maybe a s/map/mapat/ ?

File: src/libostree/ostree-repo-static-delta-core.c (hunk 1 above)
Comment (cgwalters) [inline src/libostree/ostree-repo-static-delta-core.c]: Maybe though I'm less sure about that.  There are a lot of users.  It's tempting to introduce `glnx_transfer_out_value`.  Not sure.

**Comment by cgwalters** [comment]
Sure, fixup :arrow_up:

**Comment by cgwalters** [comment]
Must have missed a `git add` or something, pushed another fixup :arrow_up:

**Comment by jlebon** [comment]
@rh-atomic-bot r+ 01fd095

**Pull request closed without merging**